ex0.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+
+def negate_range_array():
+    """
+     Using NumPy to create vector with values 0-20 and changing the sign
+     of the numbers in the range 9-15.
+     :return: The array.
+    """
+    array = np.arange(21)
```

Comment:
array is not a great variable name.
Variable names should be expressive and not generic. 
https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0